src/main/java/com/iridium/iridiumskyblock/commands/PrivateCommand.java

```diff
@@ -45,6 +45,10 @@ public boolean execute(CommandSender sender, String[] args) {
             int visitorCount = 0;
             List<IslandTrusted> islandTrusted = IridiumSkyblock.getInstance().getDatabaseManager().getIslandTrustedTableManager().getEntries(island.get());
             for (User visitor : IridiumSkyblock.getInstance().getIslandManager().getPlayersOnIsland(island.get())) {
+
+                if (island.get().getMembers().contains(visitor))
```

Comment:
Island#getMembers should be saved as a variable somewhere to be more efficient, it wont make a huge difference but like ye

Reply:
This already checks below line?

Reply:
no island.getMembers performs a binary search on the users entries to get the members, so we should be saving something like Lisit<User> islandMembers = island.get().getMemebers() to make it more efficient